exrsplit/__main__.py

```diff
@@ -123,6 +129,7 @@ def split_exr(args):
                     target_file = '{}.{}.exr'.format(exrsplit.output_file_name(layer[0]), layer[0].channel)
                 else:
                     target_file = '{}.exr'.format(exrsplit.output_file_name(layer[0]))
+                target_file = '{}_{}'.format(os.path.basename(os.path.splitext(inputfile)[0]), target_file)
```

Comment:
Any reason for the change? I'd rather keep the current behavior (seems easier to use in scripts if the output file name is just the name of the layer).

If you want to add a prefix, you can add an optional parameter to set a prefix to output files.

Reply:
I did this because I deal with exr sequences, when you habe many exr in the same directory and use exrsplit on them you overwrite the split layers but it makes sense to have an optional parameter as you suggest.